src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/LanguageServerResources.Designer.cs

```diff
@@ -0,0 +1,216 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace Microsoft.CodeAnalysis.LanguageServer {
```

Comment:
This file should not be generated. Designer files are source generated by Arcade as part of the build.